test/core/util/grpc_fuzzer.bzl

```diff
@@ -23,6 +23,7 @@ def grpc_fuzzer(name, corpus, srcs = [], deps = [], **kwargs):
     external_deps = [
       'gtest',
     ],
+    size = "enormous",
```

Comment:
out of curiosity, are they really that large? https://docs.bazel.build/versions/master/be/common-definitions.html#test.size indicates that that'd require 800 MB of memory. Maybe we want to use "timeout" instead of "size"? See https://docs.bazel.build/versions/master/be/common-definitions.html#test.timeout

Reply:
I see what you are saying. I'll use timeout instead.

Reply:
Done.